@ecomplus/storefront-components/src/html/KitProductVariations.html

```diff
@@ -0,0 +1,94 @@
+<div class="kit-product-variations">
+  <div
+    class="glide"
+    ref="glide"
+  >
+      <div
+        class="glide__track"
+        data-glide-el="track"
+      >
+        <ul class="glide__slides kit-product-variations__list">
+          <li
+            v-for="(item, index) in localItems"
+            class="glide__slide"
+          >
+            <button
+              v-if="variationKit[activeIndex]"
+              @click="removeItemFromKit(activeIndex)"
+              type="button"
+              :aria-label="i19close"
+              class="close"
+            >
+              <i class="i-times-circle"></i>
+            </button>
+            <div class="kit-product-variations__item">
+              <div class="kit-product-variations__item-head">
+                <div class="kit-product-variations__picture">
+                  <a-picture
+                    class="gallery__big-image"
+                    :src="getImg(item, null, 'normal')"
+                  />
+                </div>
+                <div class="kit-product-variations__info">
+                  <h2>{{ getName(item) }}</h2>
+                  <b>{{ i19quantity }}: 1 </b>
+                </div>
+              </div>
+
+              <slot
+                name="variations"
+              >
+                <product-variations
+                  :key="item.key"
+                  :product="item"
+                  :selected-id.sync="selectedVariationId"
+                  :max-options-btns="maxOptionsBtns"
+                  :class="variationKit[activeIndex] || variationKitReady.length !== max ? 'kit-product-variations--show' : 'kit-product-variations--hide'"
```

Comment:
```suggestion
                  :class="variationKit[activeIndex] || variationKitReady.length !== max
                    ? 'kit-product-variations--show'
                    : 'kit-product-variations--hide'"
```

Isso aqui era `!== max` mesmo? < não? Estranho

Reply:
Isso ai, faz mais sentido.